moveit_plugins/moveit_simple_controller_manager/CMakeLists.txt

```diff
@@ -26,8 +29,6 @@ catkin_package(
   CATKIN_DEPENDS moveit_core actionlib control_msgs
     )
 
-include_directories(include)
```

Comment:
No, the key is to move the "include" include directive above the "${catkin_INCLUDE_DIRS}".

The rest was just moving include directives together.